2019/index.html

```diff
@@ -95,13 +95,65 @@ <h2>Justin M. Keyes</h2>
                   <p>
                   </p>
                 </div>
-                </p>
               </div>
             </div>
           </section>
 
           <hr class="my-4">
 
+          <section id="about-vimconf2019">
+            <div class="panel text-center fat-border">
+              <div class="text-left">
+                <h4 id="開催概要">開催概要</h4>
+                <p>今年もVimConfの季節がやってまいりました。</p>
+                <p>VimConf は、世界初かつ世界で唯一のコミュニティによって定期運営されているVimの国際カンファレンスです。</p>
+                <h5 id="ゲストスピーカー">ゲストスピーカー</h5>
+                <p>今年は基調講演ゲストスピーカーとして、Vimの最たる可能性を引き連れた以下のお二方をお呼びします。</p>
+                <ul class="about">
+                  <li>prabirshrestha
+                    <ul class="about"><li>vim-lsp - author</li></ul>
+                  </li>
+                  <li>justinmk
+                    <ul class="about"><li>Neovim - 開発リーダー</li></ul>
+                  </li>
+                </ul>
+                <h5 id="vim-lspとは？">vim-lspとは？</h5>
+                <p>各エディタ（Vim・Neovim・Emacs・VSCode……etc）で統一された開発機能を利用するための仕様、「Language Server Protocol」をVim・Neovimで利用するためのプラグインです。</p>
+                <ul class="about">
+                  <li><a href="https://microsoft.github.io/language-server-protocol/">Language Server Protocolとは？</a></li>
+                  <li><a href="https://github.com/prabirshrestha/vim-lsp">prabirshrestha/vim-lsp</a></li>
+                </ul>
+                <p>これを利用することにより、その言語専用に作られた入力補完・コードフォーマットなどを、Vimから利用することができます。</p>
+                <h5 id="Neovimとは？">Neovimとは？</h5>
+                <p>高度な拡張性を与えるために分岐した、Vimのフォークです。</p>
+                <p>MessagePackやUIとエンジンが疎結合化など、Vimと比べても目立つ機能が多くあります。</p>
+                <ul class="about">
+                  <li><a href="https://neovim.io/">Neovim: hyperextensible Vim-based text editor</a></li>
+                </ul>
+                <p>GUIフロントエンドが多く存在するのは特徴的です。</p>
+                <ul class="about">
+                  <li><a href="https://www.onivim.io/">Onivim 2 (future)</a></li>
+                  <li><a href="https://github.com/akiyosi/gonvim">gonvim: Neovim GUI written in Golang, using a Golang qt backend</a></li>
+                </ul>
+                <hr>
+                <h5 id="スピーカーについて">スピーカーについて</h5>
+                上記基調講演ゲストスピーカーの他にも、以下のスピーカーの登壇を予定しております！
+                <ul class="about">
+                  <li>スピーカー（<strong>近日公開！</strong>）</li>
+                </ul>
+                <h5 id="チケット販売">チケット販売</h5>
+                <ul class="about">
+                  <li>チケット販売ページ（<strong>近日公開！</strong>）</li>
+                </ul>
+                <h5 id="その他">その他</h5>
+                <p>発表スライドはすべて英語です。発表は英語か日本語かいずれかの言語で行われます。</p>
+                <p>発表に関しては英語⇒日本語、日本語⇒英語の双方向の翻訳を予定しております。</p>
+                <hr>
+                <p>それでは、当日の来日、お待ちしております。</p>
```

Comment:
日本に住んでいる人が多くくることは明らかなので、以下のようにお願いします

```suggestion
                <p>皆様のご参加をお待ちしております。</p>
```